### DISCUSSION TITLE
Error: getaddrinfo ENOTFOUND in API route

### DISCUSSION THREAD

**Discussion by utopictown** [vercel/next.js#19634] [Help] [answered]
i got Error: getaddrinfo ENOTFOUND api.my-app.test error in API route when trying to fetch data from local backend (api.my-app.test). i also have tried using original address (localhost:5000) but no success, however it works fine when i fetch on normal page function.
fetching data from dummy API [https://jsonplaceholder.typicode.com/posts](https://jsonplaceholder.typicode.com/posts) works too. 

i use docker for my development environment, here is my stack:
nginx 80 to 80
nextjs - localhost:3000 to localhost (my-app.test)
expressjs - localhost:5000  to localhost (api.my-app.test)

thank you in advance

**Comment by utopictown** [comment, marked as answer]
i need to point my URL to be the container name instead of http address

**Reply by AusafMo** [reply]
hey! While trying to deploy a flask app onto Vercel, I am running into the same issue. I would be grateful if you could expand on how you solved this in a bit more detail (if you remember it that is)  :) 
![image](https://github.com/vercel/next.js/assets/75237046/5e60ab13-d15d-43e7-bd79-1781cdfbab68)
